efficientdet/hparams_config.py

```diff
@@ -276,6 +276,9 @@ def default_detection_configs():
   # A temporary flag to switch between legacy and keras models.
   h.use_keras_model = True
 
+  # in case of CPU memory leak bug try with True
+  h.each_epoch_in_separate_process = False
```

Comment:
Maybe make this True in default? Any drawback for having separate process?

Reply:
I don't see any major drawbacks. I agree that True in default is a good idea. Things to consider 1) it has not been tested on Windows 2) it has not been tested in cloud 3) it has not been tested with TPU. Having said that, I think it should work in all environments and there is little risk in these issues

Reply:
Many people don't know this option and still have OOM issues if it is False.
It seems safe to me, but if it causes any unexpected issues later, we can revert it back to False.

Reply:
commited h.each_epoch_in_separate_process = True in default